pkg/experimental/tests/BUILD

```diff
@@ -1,5 +1,101 @@
+# Copyright 2020 The Bazel Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
 load(":pkg_filegroup_test.bzl", "pkg_filegroup_analysis_tests", "pkg_filegroup_unit_tests")
+load("@rules_pkg//experimental:pkg_filegroup.bzl", "pkg_filegroup", "pkg_mkdirs")
+load("@rules_pkg//experimental:rpm.bzl", "pkg_rpm")
+load("@rules_python//python:defs.bzl", "py_test")
 
 pkg_filegroup_analysis_tests()
 
 pkg_filegroup_unit_tests()
+
+filegroup(
+    name = "ars",
+    srcs = glob(["testdata/*.ar"]),
+)
+
+pkg_filegroup(
+    name = "ars_pfg",
+    prefix = "/test",
+    srcs = [
+        ":ars",
+    ],
+    attrs = {"unix": ["0755", "root", "root"]},
+)
+
+pkg_mkdirs(
+    name = "var_log_foo",
+    dirs = ["/var/log/foo"],
+    attrs = {"unix": ["0755", "root", "root"]},
+)
+
+pkg_rpm(
+    name = "test_rpm",
+    data = [":ars_pfg", "var_log_foo"],
+    version = "1.1.1",
+    release = "2222",
+    license = "Apache 2.0",
+    architecture = "noarch",
+    summary = "pkg_rpm test rpm summary",
+    description = """pkg_rpm test rpm description""",
```

Comment:
Sorry it took me so long to get back to you on this.  Been rather busy with internal stuff.

This is convention/force-of-habit for me, since RPM descriptions are typically multi-line.  I'm happy to change it to a single-quoted string, though.